### PR TITLE
test: Add a logger to print the instance id.

### LINKED ISSUE #1128: Add a log in the asserts for system tests on clusters

### PR DESCRIPTION
**Pull request by danieljbruce**
This PR is used to help with debugging cluster issues.

Fixes #1128

### CODE REVIEW

**Review by kolea2 — APPROVED** [review]
please rename PR to "test: ..." prefix

File: system-test/cluster.ts [hunk 1]
```diff
@@ -67,6 +67,7 @@ describe('Cluster', () => {
         time_created: Date.now(),
       },
     });
+    console.log(`Test Instance Id: ${instanceId}`);
     await operation.promise();
```
Comment (shantstepanian) [inline system-test/cluster.ts:71]: Can we log the cluster we query in checkMetadata (e.g. method starting line 40)?

File: system-test/cluster.ts [hunk 2]
```diff
@@ -67,6 +67,7 @@ describe('Cluster', () => {
         time_created: Date.now(),
       },
     });
```
Comment (shantstepanian) [inline system-test/cluster.ts:69]: Is it also possible to add more IDs on the cluster that is being compared in the assertion? i.e. to easily show which cluster is mismatching the ID

```
  1) Cluster << would be nice to have the cluster ID here >>
       Update cluster
         Starting from autoscaling
           should change cluster to manual scaling:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:

3 !== 5

      + expected - actual

      -3
      +5

      at checkMetadata (build/system-test/cluster.js:31:16)
          -> /workspace/system-test/cluster.ts:42:12
      at async Context.<anonymous> (build/system-test/cluster.js:273:17)
          -> /workspace/system-test/cluster.ts:306:9
```

Comment (danieljbruce) [reply system-test/cluster.ts:69]: Hmm. I'm not sure how to make a dynamic title in the mocha framework, but we can definitely print out the cluster id.

File: system-test/cluster.ts (hunk 1 above)
Comment (danieljbruce) [reply system-test/cluster.ts:71]: Sure. We can log the cluster id.

File: system-test/cluster.ts (hunk 2 above)
Comment (shantstepanian) [reply system-test/cluster.ts:69]: No worries, yea, whatever is easier to do, as I'm not too familiar w/ Typescript and its frameworks. So long as we see the cluster ID logged earlier in the log that corresponds w/ the test failure, we should be alright

**Pull request merged**